tests/languages/node_test.py

```diff
@@ -126,12 +126,56 @@ def _make_hello_world(tmp_path):
     )
 
 
+def _make_typescript_hello_world(tmp_path):
+    package_json = """
+{
+  "name": "t",
+  "version": "0.0.1",
+  "bin": {
+    "node-hello": "./bin/main.js"
+  },
+  "scripts": {
+    "prepare": "tsc",
+    "build": "tsc"
+  },
+  "devDependencies": {
+    "typescript": "^5.3.0"
+  }
```

Comment:
this is going to make the test prohibitively slow -- can you make a faster example?

Reply:
@asottile I took an example from https://github.com/pre-commit/pre-commit/pull/3259 instead. Will that work?